### PR TITLE
Added AddThingsToDoAd example and renamed the directory accordingly.

### PR DESCRIPTION
**Pull request by fiboknacky**
Based heavily on AddHotelAd.

### CODE REVIEW

File: examples/HotelAndTravel/AddThingsToDoAd.php [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+     */
+    private static function addCampaignBudget(GoogleAdsClient $googleAdsClient, int $customerId)
+    {
+        // Creates a campaign budget.
+        $budget = new CampaignBudget([
+            'name' => 'Interplanetary Cruise Budget #' . Helper::getPrintableDatetime(),
+            'delivery_method' => BudgetDeliveryMethod::STANDARD,
+            // Sets the amount of budget.
+            'amount_micros' => 50000000,
+            // Makes the budget explicitly shared.
```
Comment (jradcliff) [inline examples/HotelAndTravel/AddThingsToDoAd.php:161]: Is this a requirement for Things to do campaigns?

File: examples/HotelAndTravel/AddThingsToDoAd.php [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+        string $campaignResourceName
+    ) {
+        // Creates an ad group.
+        $adGroup = new AdGroup([
+            'name' => 'Earth to Mars Cruise #' . Helper::getPrintableDatetime(),
+            // Sets the campaign.
+            'campaign' => $campaignResourceName,
+            // Sets the ad group type to TRAVEL_ADS. This cannot be set to other types.
+            'type' => AdGroupType::TRAVEL_ADS,
+            'cpc_bid_micros' => 10000000,
```
Comment (jradcliff) [inline examples/HotelAndTravel/AddThingsToDoAd.php:270]: Just double-checking: does setting this when the bidding strategy is MaximizeConversionValue make sense? Is it just capping the max bid?

File: examples/HotelAndTravel/AddThingsToDoAd.php [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+     * @param string $adGroupResourceName the resource name of ad group that a new ad group ad will
+     *     belong to
+     */
+    // [START add_things_to_do_ad_3]
+    private static function addAdGroupAd(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $adGroupResourceName
+    ) {
+        // Creates a new ad group ad and set a travel ad info.
```
Comment (jradcliff) [inline examples/HotelAndTravel/AddThingsToDoAd.php:308]: nit: 'set' => 'sets'

File: examples/HotelAndTravel/AddHotelAd.php [hunk 4]
```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-namespace Google\Ads\GoogleAds\Examples\HotelAds;
+namespace Google\Ads\GoogleAds\Examples\HotelAndTravel;
```
Comment (jradcliff) [inline examples/HotelAndTravel/AddHotelAd.php:19]: WDYT of just `Travel` instead?

File: examples/HotelAndTravel/AddThingsToDoAd.php (hunk 1 above)
Comment (fiboknacky) [reply examples/HotelAndTravel/AddThingsToDoAd.php:161]: Yes. Confirmed.

File: examples/HotelAndTravel/AddThingsToDoAd.php (hunk 2 above)
Comment (fiboknacky) [reply examples/HotelAndTravel/AddThingsToDoAd.php]: Thanks for catching. I think it makes sense to add it here either. The code works without this.

File: examples/HotelAndTravel/AddHotelAd.php (hunk 4 above)
Comment (fiboknacky) [reply examples/HotelAndTravel/AddHotelAd.php]: That's fine too. I just checked and it seems we use Travel to cover both Hotel and Travel now.

File: examples/HotelAndTravel/AddThingsToDoAd.php (hunk 1 above)
Comment (jradcliff) [reply examples/HotelAndTravel/AddThingsToDoAd.php]: Could you add a comment so readers know `false` is not an option?

Comment (fiboknacky) [reply examples/HotelAndTravel/AddThingsToDoAd.php]: Done.

**Review by jradcliff — APPROVED** [review]

**Review by Raibaz — APPROVED** [review]

**Pull request merged**